force-app/main/freelanceCommunity/lwc/hot_frilanstolkThreadList/hot_frilanstolkThreadList.js

```diff
@@ -0,0 +1,447 @@
+import { LightningElement, wire, track } from 'lwc';
+import { NavigationMixin } from 'lightning/navigation';
+import getMyThreads from '@salesforce/apex/HOT_ThreadListController.getMyThreads';
```

Comment:
Kan du ikke bruke ulike metoder for bruker og frilanstolk så du slipper mindre filtrerer her, eller er det det samme?

Reply:
prøver på det